cognite_toolkit/_cdf_tk/cdf_toml.py

```diff
@@ -22,7 +22,6 @@
 class CLIConfig:
     default_organization_dir: Path
     default_env: str = "dev"
-    feature_flags: dict[str, bool] = field(default_factory=dict)
```

Comment:
What is the thinking behind moving this to its own category? (I am not against it, just curious)

Reply:
Just aesthetics/clarity and simplicity! Didn't like the way we got `CDFToml.load().cdf.feature_flags` because one may think it was a feature flag of CDF, not the Toolkit.